### PR TITLE
Conv fixes

### PR DESCRIPTION
**Pull request by MaheshRavishankar**

### CODE REVIEW

File: iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp [hunk 1]
```diff
@@ -710,12 +710,6 @@ void ConvertToGPUPass::runOnFunction() {
     PartitionPLoopToWorkgroups, RemoveLinalgRange>(context);
   // clang-format on
 
-  patterns.insert<MapLinalgOpToGlobalInvocationId<linalg::ConvOp>>(context);
-  if (useLegacyConvLowering)
```
Comment (antiagainst) [inline iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp:714]: Is it also possible to eliminate the command line option now?

Comment (MaheshRavishankar) [reply iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp:714]: Yes, I will do it. Its WIP. I created a PR just to import it into internal google tooling to check some previously failing internal tests. Those are still failing (cant repro it anywhere else though). I will have to wait for Hanhan's tooling.

**Comment by hanhanW** [comment]
I think this should be merged into `google:main`, not `google:google`?

**Comment by MaheshRavishankar** [comment]
Sorry, the message doesnt say that (I will fix that). This PR is WR. I am using this PR to pull this CL into google infra for internal testing. No need to review.

**Comment by antiagainst** [comment]
You probably want to mark it as "draft" then and flip the bit when it's ready for review. :)

**Comment by benvanik** [comment]
this still valid?

**Comment by MaheshRavishankar** [comment]
No not really. The change would different now. Closing this

**Pull request closed without merging**